### ISSUE TITLE
Where are the Elm communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/elm#217]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Elm supported by one or more large organizations?
* Does Elm have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Elm? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Elm? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Elm? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Elm?
* Are there popular newsletters for Elm?
* Is Elm taught at programming bootcamps? (If so, what are some examples?)
* Is Elm taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Elm?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by tuxagon** [comment]
> Is Elm supported by one or more large organizations?

Elm is maintained and supported by Evan Czaplicki with support from NoRedInk, an organization in San Franscisco that hired Evan full-time to work on Elm.

> Does Elm have an official community manager?

I'm not sure, but there are admins in the slack community that could be reached out to as community managers. They follow the benevolent dictator mindset with regards to development, but as for community, I'm not aware of a specific person to point to.

TL;DR: Evan, NoRedInk, Slack Admins most likely

> Do you know of specific communities (online or offline) that are enthusiastic about Elm? (Chat communities, forums, meetups, student clubs, etc)

This is an intersting question because I'd think any community around a language would be enthusiastic about a language, so I'll just list the communities around elm 😄

* [Elm Slack](https://elmlang.slack.com/) ([Join](https://elmlang.herokuapp.com/))
* [Elm Discourse](https://discourse.elm-lang.org/)
* [Elm Subreddit](https://www.reddit.com/r/elm/)
* [Elm Meetups](https://www.elmlog.com/meetup_groups)

> Are there popular conferences for Elm? (If so, what are some examples?)

* [Elm Conference](https://www.elm-conf.us/)
* [Elm Europe](https://2018.elmeurope.org/)

> Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Elm? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

I've put feelers out to the community to answer this question. Right now, I don't know of any.

> Are there specific groups or programs dedicated to mentoring people in Elm?

There are courses people offer online to teach elm and meetup groups in various places, but I'm not aware of any place designed to mentor.

> Are there popular newsletters for Elm?

I'm not aware of any newsletters where you get updates passively like that. They may exist, but I've not encountered them. A good general place to find stuff out about elm would probably be slack or discourse

* [Elm Slack](https://elmlang.slack.com/) ([Join](https://elmlang.herokuapp.com/))
* [Elm Discourse](https://discourse.elm-lang.org/)

> Is Elm taught at programming bootcamps? (If so, what are some examples?)

There are courses online, but no bootcamps I'm aware of that will train and then help place you in a job.

> Is Elm taught at universities? (If so, what are some examples?)

I've heard of some who have mentioned this before, but I don't know of any active, recurring classes. They are usually one-off classes I've seen with regards to covering functional programming. The only link I found that led to a university is

* https://www.classes.cs.uchicago.edu/archive/2015/winter/22300-1/

but that is 3 years old.

Hopefully helpful resources:

* https://github.com/elm-lang
* http://elm-lang.org:1234/community
* https://www.noredink.com/
* https://ellie-app.com/new (the live elm editor)

---

I hope this helps! Let me know if I can answer anything else.

**Comment by kytrinyx** [comment]
This is such great information, thank you!

**Comment by tuxagon** [comment]
@kytrinyx I received an answer about organizations that teaches elm to underrepresented groups.

https://github.com/elmbridge

**Comment by neganp** [comment]
There's also a pretty regular podcast: https://elmtown.audio/

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. (Should you come across any other good sources, feel free to add them here, I'll be coming back to this). ❤️ 💛 💚 💙 💜 🖤 Thanks!
